File: Dataset/Leetcode/valid/66/138.cpp
 class Solution {
public:
    vector<int> XXX(vector<int>& digits) {
        for (int i = digits.size() - 1; i >= 0; i--) {
            digits[i]++ ;
            digits[i] = digits[i] <= 9 ? digits[i] : 0;
            // 如果没有进位直接 return
            if (digits[i]) return digits;
        }
        // 走出for循环都没有return，证明最高位有进位，insert一个1 再 return
        digits.insert(digits.begin(), 1);
        return digits;
    }
};

File: Dataset/Leetcode/train/62/138.py
 class Solution:
    def XXX(self, m: int, n: int) -> int:
        path_list = [1]*m
        for i in range(1, n):
            for j in range(1,m):
                path_list[j]+=path_list[j-1]
        return path_list[-1]

